psng/python/tool_measurement.py

```diff
@@ -164,8 +163,17 @@ def on_spbtn_block_height_value_changed(self, gtkspinbutton, data=None):
         i.set_line(0)
         self.buffer.insert(i, "%s \n" % c)
 
+    # Down probe to table for measuring it and use for calculate tool setter height and can set G10 L20 Z0 if you tick auto zero
+    def on_btn_probe_table_released(self, gtkbutton, data=None):
+        # Start psng_probe_table.ngc
+        if self.ocode("o<psng_probe_table> call") == -1:
+            return
+        a = self.probed_position_with_offsets()
+        self.add_history(gtkbutton.get_tooltip_text(), "Z", z=a[2])
+        self.set_zerro("Z", 0, 0, a[2])
+        
     # Down probe to tool setter for measuring it vs table probing result
-    def on_btn_probe_tool_setter_released(self, gtkbutton, data=None):
+    def on_btn_probe_setter_released(self, gtkbutton, data=None):
```

Comment:
Why rename this? `probe_tool_setter` is IMO clearer, and now there is a mix of both `probe_tool_setter` and `probe_setter` in the code.